lib/cli/config.js

```diff
@@ -69,7 +69,7 @@ const parsers = (exports.parsers = {
  */
 exports.loadConfig = filepath => {
   let config = {};
-  debug(`loadConfig: "${filepath}"`);
+  debug('loadConfig: "%s"', filepath);
```

Comment:
I would suggest to change `line 98` as well to `debug('findConfig: found "%s"', filepath);`

Reply:
done